year-pages/2019-conferences-list.md

```diff
@@ -3,4 +3,5 @@
 | Conference | Date | Venue | Description | Scholarship |
 |------------|------|-------|-------------|-------------|
 | [RubyConf India](https://www.rubyconfindia.org/) | 20-21 Jan | Resort Rio and Club Rio Royale, Goa | A global event complementing other RubyConf events across the world. This year it's a single track 2 day event, focused on the Language, Framework and Tools. | [Yes](https://www.rubyconfindia.org/scholarship/) |
-| [JSFoo](https://jsfoo.in/2019-pune/) | 10 Jan | MCCIA Trade Tower, Pune | The second Pune edition of India’s premier JavaScript conference. | No
\ No newline at end of file
+| [JSFoo](https://jsfoo.in/2019-pune/) | 10 Jan | MCCIA Trade Tower, Pune | The second Pune edition of India’s premier JavaScript conference. | No
+| [SACON 2019](https://www.sacon.io/) | 15-16 Feb | Banglore |  India’s 1st Security Architecture Conference, to solve this competency gap. | No
```

Comment:
- Please remove the "2019" from the title. It is implicit. :)
- In the description, just - `India’s first conference about Security Architecture.`

The list is in reverse chronological order. So this entry needs to be on the top.

Reply:
Please make the changes in description too as mentioned above.